chain/chain/src/store/mod.rs

```diff
@@ -239,7 +239,7 @@ pub trait ChainStoreAccess {
 
             if shard_layout != prev_shard_layout {
                 let parent_shard_id = shard_layout.get_parent_shard_id(shard_id)?;
-                tracing::debug!(
+                tracing::info!(
```

Comment:
Why info here?

Reply:
Because it is very rare event which seems important though. I would log everything related to resharding through info.
We don't have `RUST_LOG=debug` on all nodes, because it may impact performance, so in the case of incident we can end up with only info logs on main nodes.